src/openrct2/drawing/Drawing.h

```diff
@@ -525,6 +526,7 @@ void gfx_unload_csg();
 const rct_g1_element* gfx_get_g1_element(ImageId imageId);
 const rct_g1_element* gfx_get_g1_element(ImageIndex image_id);
 void gfx_set_g1_element(ImageIndex imageId, const rct_g1_element* g1);
+std::optional<rct_gx> GxfLoadGx(const std::vector<uint8_t>& buffer);
```

Comment:
I read this so many times and somehow never noticed gfx is spelt wrong. GfxLoadGx